packages/cycle-scripts-one-fits-all/template/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compileOnSave": false,
     "compilerOptions":
     {
-        "module": "ES6",
```

Comment:
I changed as Code was complaining - it seems it's intellisense is a bit muddled wrt to case, even though it uses the typings